Config revamp

erezmus opened the pull request.

diversemix commented on src/config.ts:
```diff
@@ -33,15 +34,43 @@ export interface ClientPublicConfig {
 }
 
-const configPath = process.env.CONFIG_PATH ? process.env.CONFIG_PATH : '/etc/reviewer/config.json';
+// const configPath = process.env.CONFIG_PATH ? process.env.CONFIG_PATH : '/etc/reviewer/config.json';
```
Do we want to delete this line?

diversemix commented on src/config.ts:
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/camelcase */
```
Why?

erezmus commented:
@erkannt ok all done now. I've made it more closely aligned to the reviewer-client way of doing things.
can jump on call to explain if needed.

I've also changed `make start` to `make start_dev` to make more consistent with client. I've also added `make start_test` so one can do integration tests while still running in development mode
